tests/sdk/conftest.py

```
@@ -0,0 +1,73 @@
+    Returns a Honcho SDK client configured to talk to the test API.
+    """
+    http_client = httpx.Client(
+        transport=client._transport,  # pyright: ignore
+        base_url=str(client.base_url),
+        headers=client.headers,
+    )
+
+    honcho_client = Honcho(
+        workspace_id="sdk-test-workspace-sync", http_client=http_client
+    )
+    yield honcho_client
```

Ruff (0.11.9)

No teardown in fixture `honcho_sync_test_client`, use `return` instead of `yield`

Replace `yield` with `return`

(PT022)